client/src/project/ProjectStatus.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div v-if="event && status">
+    <div class="row">
+      <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 130.2 130.2">
+        <circle class="path circle" fill="none" stroke="#73AF55" stroke-width="6" stroke-miterlimit="10" cx="65.1" cy="65.1" r="62.1"/>
+        <polyline class="path check" fill="none" stroke="#73AF55" stroke-width="6" stroke-linecap="round" stroke-miterlimit="10" points="100.2,40.2 51.5,88.8 29.8,67.5 "/>
+      </svg>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div v-if="status === 'verified'" >
+          <h1>Your project has been confirmed.</h1>
+          <h2 class="text-center">See you on {{formattedDate}}!</h2>
+        </div>
+        <div v-else>
```

Comment:
I'd be inclined to use `v-else-if="status === 'cancelled'"` here just so if someone fudges with the URL that they don't end up seeing cancelled show up.

Reply:
Fair